safedelete/tests/test_soft_delete.py

```diff
@@ -116,10 +113,16 @@ def test_undelete_queryset(self):
         self.assertEqual(SoftDeleteModel.objects.count(), 0)
 
         SoftDeleteModel.objects.all().undelete()  # Nonsense
-        self.assertEqual(SoftDeleteModel.objects.count(), 0)
+        self.assertEqual(SoftDeleteModel.objects.count(), 0,
+                         f"When did 'SoftDeleteModel.objects.all().undelete()', "
+                         f"expected 0 SoftDeleteModel.objects.count, got {SoftDeleteModel.objects.count()} instead.\n"
+                         f"SoftDeleteModel.objects.all(): {SoftDeleteModel.objects.all()}\n"
+                         )
 
         SoftDeleteModel.deleted_objects.all().undelete()
```

Comment:
Specifically, I'm guessing this is where it breaks and you need to refresh deleted_objects, but I haven't run the code

Reply:
it falls when trying to:
`SoftDeleteModel.objects.all().undelete()
self.assertEqual(SoftDeleteModel.objects.count(), 0,` - it is really weird for me.